doc/start/hardware-recommendations.rst

```diff
@@ -110,8 +110,9 @@ option.
    needed, depending on the exact configuration of the system.
 
 When using the legacy FileStore backend, the page cache is used for caching
-data, so no tuning is normally needed, and the OSD memory consumption is
-generally related to the number of PGs per daemon in the system.
+data, so no tuning is normally needed. When using the legacy FileStore backend,
+the OSD memory consumption is related to the number of PGs per daemon in the
+system.
```

Comment:
Suggest `When using the legacy Filestore back end, OSD memory consumption is related to the number of PGs per daemon.`

When my book was in the editing phase, there was some controversy re `backend` and `back end`; the conclusion was to prefer the latter.

Reply:
Alright.